lib/management/poll.js

```diff
@@ -97,6 +97,9 @@ function getStats() {
 }
 
 function pushStats(managementEndpoint, instanceId, remoteToken, next) {
+    if (process.env.PUSH_STATS === 'false') {
+        return;
```

Comment:
Shouldn't we call `next()` even if doing nothing?

Reply:
There seems to be no scenario in which we are using `next`.

Reply:
Looks confusing but okay.